sanitiser/_geo.js

```diff
@@ -78,15 +81,15 @@ function sanitize_bbox( clean, param ) {
  * @param {bool} latlon_is_required
  */
 function sanitize_coord( coord, clean, param, latlon_is_required ) {
-  var value = parseFloat( param, 10 );
+  var value = parseFloat( param );
   if ( !isNaN( value ) ) {
-    if( lat_lon_checks[coord].is_invalid( value ) ){
-      throw new Error( 'invalid ' + lat_lon_checks[coord].error_msg );
+    if( coord === 'lat' && check_lat.is_invalid( value ) ){
```

Comment:
don't do this check anymore. if `lon` isn't being validated, `lat` doesn't need to be either.